docs/api/covidcast-signals/nssp.md

```diff
@@ -0,0 +1,99 @@
+
+## Overview
+
+[The National Syndromic Surveillance Program (NSSP)](https://www.cdc.gov/nssp/php/about/index.html) is an effort to track epidemiologically relevant conditions.
+This dataset in particular tracks emergency department (ED) visits arising from a subset of influenza-like illnesses, specifically influenza, COVID-19, and RSV.
+It is derived from the CDC's [Respiratory Virus Response NSSP Emergency Department Visit Trajectories dataset](https://data.cdc.gov/Public-Health-Surveillance/2023-Respiratory-Virus-Response-NSSP-Emergency-Dep/rdmq-nq56/about_data), which started reporting data in late 2022.
+As of May 2024, NSSP received data from 78% of US EDs.
+
+| Signal                          | Description                                                             |
+|---------------------------------|-------------------------------------------------------------------------|
+| `pct_visits_covid`              | Percent of ED visits that had a discharge diagnosis code of covid              |
+| `pct_visits_influenza`          | Percent of ED visits that had a discharge diagnosis code of influenza          |
+| `pct_visits_rsv`                | Percent of ED visits that had a discharge diagnosis code of rsv                |
+| `pct_visits_combined`           | Percent of ED visits that had a discharge diagnosis code of covid, influenza, or rsv |
+| `smoothed_pct_visits_covid`     | 3 week moving average of `pct_visits_covid`                        |
+| `smoothed_pct_visits_influenza` | 3 week moving average of `pct_visits_influenza`                    |
+| `smoothed_pct_visits_rsv`       | 3 week moving average of `pct_visits_rsv`                          |
+| `smoothed_pct_visits_combined`  | 3 week moving average of `pct_visits_combined`                     |
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Estimation
+
+The percent visits signals are calculated as a fraction of visits at facilities reporting to NSSP, rather than all facilities in the area.
+County and state level data is reported as-is from NSSP, without modification, while `hrr` and `msa` are estimated by Delphi.
+
+### Geographic weighting
+As the original data is a percentage and raw case counts are not available, `hrr` and `msa` values are computed from county-level data using a weighted mean. Each county is assigned a weight equal to its population in the last census (2020).
+This assumes that the number of ED visits is proportional to the overall population of a county, i.e. the per-capita ED visit rate is the same for all counties, which may not be the case (for example, denser counties may have easier access to EDs and thus higher rates of ED visits per capita).
+
+State-level data is reported separately, and is **not** simply an average of the county-level data, but may contain facilities omitted at the regional level (for example, if small facilities are excluded for privacy reasons).[^1] 
```

Comment:
[The source](https://data.cdc.gov/Public-Health-Surveillance/2023-Respiratory-Virus-Response-NSSP-Emergency-Dep/rdmq-nq56/about_data) is a mix of state and county level data. There is a separate state-only source that we are not using, which is the one that appears to be a rounded version of this one.

That for example should probably be a bit more involved. There are entire states which don't have local data; I suspect it's because the state itself is reporting, even if the local ERs/counties aren't, but they're only providing state-level data.

Compare the state and county level reporting in [this set of graphs](https://github.com/cmu-delphi/covidcast-indicators/pull/1952#issuecomment-2090894314)